teacher.rb

```diff
@@ -0,0 +1,13 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
```

Comment:
- [ ] Please read my above comments, whatever I mentioned is also happening here too. Kindly fix the issue of the arguments for the person using the child or extended class.